### PR TITLE
Switch to using native `traceback`

### PR DESCRIPTION
**Pull request by galipremsagar**
## Description
This PR switches pytest traceback to `native` instead of prettified pytest traceback that takes longer to finish and spits out the source code of the file where the error happens too which is not needed given the time savings.

With pytest traceback:
Screenshot 2024-09-19 at 2 34 57 PM
Screenshot 2024-09-19 at 2 35 07 PM
Screenshot 2024-09-19 at 2 35 20 PM

With `native` traceback:
Screenshot 2024-09-19 at 2 34 04 PM

## Checklist
- [ ] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.

### CODE REVIEW

**Review by bdice — APPROVED** [review]
This is fine with me. We could alternatively set this with the config options:
```
[pytest]
addopts = --tb=native
```

**Comment by galipremsagar** [comment]
> This is fine with me. We could alternatively set this with the config options:
> 
> ```
> [pytest]
> addopts = --tb=native
> ```

Move it to config options.

File: ci/test_wheel_dask_cudf.sh [hunk 1]
```diff
@@ -41,6 +41,7 @@ pushd python/dask_cudf/dask_cudf
 DASK_DATAFRAME__QUERY_PLANNING=True python -m pytest \
   --junitxml="${RAPIDS_TESTS_DIR}/junit-dask-cudf.xml" \
   --numprocesses=8 \
+  --dist=worksteal \
```
Comment (rjzamora) [inline ci/test_wheel_dask_cudf.sh:44]: Curious how much of a speedup you see with this change?

**Review by rjzamora — APPROVED** [review]
Thanks @galipremsagar - Seems fine on the dask side.

File: ci/test_wheel_dask_cudf.sh (hunk 1 above)
Comment (galipremsagar) [reply ci/test_wheel_dask_cudf.sh:44]: When we switched cudf wide, we were able to cut down 25% of the execution time: https://github.com/rapidsai/cudf/pull/15207

**Pull request merged**